src/util-byte.h

```diff
@@ -226,7 +226,7 @@ int ByteExtractStringInt64(int64_t *res, int base, size_t len, const char *str);
  * \return n Number of bytes extracted on success
  * \return -1 On error
  */
-int ByteExtractStringInt32(int32_t *res, int base, size_t len, const char *str);
+int  ByteExtractStringInt32(int32_t *res, int base, size_t len, const char *str);
```

Comment:
I think a `WARN_UNUSED` was missed here and was present in the previous PR.

Reply:
🫣 Thanks for catching this.